tests/application/cms/test_page_hierarchy_models.py

```diff
@@ -0,0 +1,25 @@
+def test_references_between_subtopic_and_topic(stub_topic, stub_subtopic):
+    assert stub_topic.subtopics == [stub_subtopic]
+    assert stub_subtopic.topic == stub_topic
```

Comment:
maybe not 🤷‍♂️

Reply:
How do you mean? Subtopics only belong to one topic.

Reply:
Sorry, this is in ref to my earlier comment: "I think you need back_populates on both sides of the relationship for it to be bidirectionally useable in Python"